pkg/neorpc/result/version.go

```diff
@@ -39,8 +39,18 @@ type (
 		MemoryPoolMaxTransactions   int
 		ValidatorsCount             byte
 		InitialGasDistribution      fixedn.Fixed8
+
+		// Below are NeoGo-specific extensions to the protocol that are
+		// returned by the server in case they're enabled.
```

Comment:
Maybe it's worth to mention these extensions in our RPC doc?

Reply:
Added that.